### PR TITLE
Added internal tests to the run script

### PR DESCRIPTION
**Pull request by roigcarlo**
This adds the internal tests to the run_script.py script. Internal tests always run.

### CODE REVIEW

**Comment by pooyan-dadvand** [comment]
By internal tests you mean cpp unit tests?

**Comment by roigcarlo** [comment]
Yes

File: kratos/python_scripts/run_tests.py [hunk 1]
```diff
@@ -297,6 +297,16 @@ def main():
             signalTime
         )
 
+    sys.stderr.flush()
+
+    # Run the internal tests (does the same as run_cpp_tests.py)
+    print('Running internal tests', file=sys.stderr)
+    try:
+        Tester.SetVerbosity(Tester.Verbosity.TESTS_OUTPUTS)
```
Comment (pooyan-dadvand) [inline kratos/python_scripts/run_tests.py]: I would use menos verbosity like test lists or even less

**Review by pooyan-dadvand — CHANGES_REQUESTED** [review]
Does it imports applications for calling their tests?

File: kratos/python_scripts/run_tests.py (hunk 1 above)
Comment (roigcarlo) [reply kratos/python_scripts/run_tests.py]: FAILED_TESTS_OUTPUTS?

Comment (pooyan-dadvand) [reply kratos/python_scripts/run_tests.py]: I would go for PROGRESS which is similar to the python ones showing "." for each test and error messages for the failed.

**Comment by roigcarlo** [comment]
@pooyan-dadvand Done!

**Review by pooyan-dadvand — CHANGES_REQUESTED** [review]
Please change the internal test to cpp tests as they are called in wiki

File: kratos/python_scripts/run_tests.py [hunk 2]
```diff
@@ -299,13 +299,13 @@ def main():
 
     sys.stderr.flush()
 
-    # Run the internal tests (does the same as run_cpp_tests.py)
-    print('Running internal tests', file=sys.stderr)
+    # Run the internal cpp (does the same as run_cpp_tests.py)
```
Comment (pooyan-dadvand) [inline kratos/python_scripts/run_tests.py]: I suppose it is Run the cpp tests

Comment (roigcarlo) [reply kratos/python_scripts/run_tests.py]: yep already fixed in cb6688a

Comment (pooyan-dadvand) [reply kratos/python_scripts/run_tests.py]: So please change the comment also! 😃

**Review by pooyan-dadvand — APPROVED** [review]

**Comment by pooyan-dadvand** [comment]
Just update the comment!

**Comment by roigcarlo** [comment]
@pooyan-dadvand Is there any other comment wrong?

**Comment by pooyan-dadvand** [comment]
Sorry, I was checking an outdated version! 👍

**Pull request merged**